src/components/NavBar.jsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { Tab, Box } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import { Button } from "@mui/base";
+
+export default function NavBar() {
+  const navigate = useNavigate();
+  return (
+    <Box sx={{ width: "100%" }}>
+      <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
+        <Tab label="Home" onClick={() => navigate("/")} />
+        <Tab label="Dashboard" onClick={() => navigate("/dashboard")} />
+        <Tab label="Log in/Sign Up" onClick={() => navigate("/login-signup")} />
```

Comment:
I am thinking about this, and we might need to have an authenticated state. If you prefer I could take a look at this when I work on the logout task or you could work on it but spin off a separate task. I think it would be best to merge this PR asap to allow for us to more easily work on other tasks.

Reply:
I was going to suggest making this another PR as well. I think having the NavBar is nice so we can navigate around easier, but it does still need some work.